app/models/course.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Course < ApplicationRecord
+  belongs_to :schedule
```

Comment:
Shouldn't this also have a has_many for sections?

Reply:
your correct I have added it